python/pyspark/ml/image.py

```diff
@@ -168,7 +167,7 @@ def readImages(self, path, recursive=False, numPartitions=-1,
         :param bool dropImageFailures: Drop the files that are not valid images
         :param float sampleRatio: Fraction of the images loaded
         :param int seed: Random number seed
-        :rtype DataFrame: DataFrame with a single column of "images",
+        :return: DataFrame: DataFrame with a single column of "images",
```

Comment:
`:return: a DataFrame with a single column of "images",`

Reply:
done